stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/StreamFileUtil.kt

```diff
@@ -54,4 +59,127 @@ public object StreamFileUtil {
             null
         }
     }
+
+    /**
+     * Creates a Stream cache directory if one doesn't exist already.
+     *
+     * @param context The [Context] necessary to perform
+     * file operations.
+     * @param onError Called when an error occurs.
+     *
+     * @return The Stream cache directory if it exists or its creation
+     * was successful, false otherwise.
+     */
+    @Suppress("TooGenericExceptionCaught")
+    private fun getOrCreateStreamCacheDir(
+        context: Context,
+        onError: (exception: Exception) -> Unit = {},
+    ): File? {
+        return try {
+            val streamCacheDir = File(context.cacheDir, streamCacheDirName)
+            streamCacheDir.mkdir()
```

Comment:
```suggestion
            streamCacheDir.mkdirs()
```

Reply:
Implemented.

That's a really good suggestion.
I was not aware of the existence of `mkdirs()` and the difference between the two.

Thank you!